### PR TITLE
Create a stub of the file for san diego

### LINKED ISSUE #37: create a new file for annadamson hometown
## Add file for San Diego CA
- [x] create a branch
- [x] add my file
- [x] create a commit
- [x] open a pull request
- [x] have a discussion about the pull request
- [x] merge the pull request

### PR DESCRIPTION
**Pull request by annadamson**
Added sub sections for what we want to know about San Diego, CA 

This Pull Request closes #37

### CODE REVIEW

File: places/sandiego.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## San Diego, CA
+### Great Places To Eat
```
Comment (annadamson) [inline places/sandiego.md:3]: @annadamson I :heart: italian, any recommendations?

Comment (reversaw) [inline places/sandiego.md:3]: @annadamson What's the best Indian food in San Diego?

Comment (annadamson) [inline places/sandiego.md:3]: I actually do not know ... I've only had it catered ... never been to an Indian restaurant here!

**Comment by reversaw** [comment]
@annadamson Looks good 🚢

**Pull request merged**